sdk/cognitiveservices/cognitiveservices-textanalytics/review/cognitiveservices-textanalytics.api.md

```diff
@@ -0,0 +1,346 @@
+}
+
+// @public (undocumented)
+export interface ExtractKeyPhrasesErrorResult extends TextAnalysisErrorResult {
+}
+
+// @public (undocumented)
+export interface ExtractKeyPhrasesOptions extends TextAnalyticsClientKeyPhrasesOptionalParams {
+}
+
+// @public (undocumented)
+export type ExtractKeyPhrasesResult = ExtractKeyPhrasesSuccessResult | ExtractKeyPhrasesErrorResult;
+
+// @public (undocumented)
+export interface ExtractKeyPhrasesResultCollection extends Array<ExtractKeyPhrasesResult> {
+    modelVersion: string;
+    statistics?: RequestStatistics;
+}
+
+// @public (undocumented)
+export interface ExtractKeyPhrasesSuccessResult extends TextAnalysisSuccessResult {
+    keyPhrases: string[];
+}
+
+// @public (undocumented)
+export interface ExtractLinkedEntitiesErrorResult extends TextAnalysisErrorResult {
+}
+
+// @public (undocumented)
+export type ExtractLinkedEntitiesResult = ExtractLinkedEntitiesSuccessResult | ExtractLinkedEntitiesErrorResult;
+
+// @public (undocumented)
+export interface ExtractLinkedEntitiesResultCollection extends Array<ExtractLinkedEntitiesResult> {
+    modelVersion: string;
+    statistics?: RequestStatistics;
+}
+
+// @public (undocumented)
+export interface ExtractLinkedEntitiesSuccessResult extends TextAnalysisSuccessResult {
+    readonly entities: LinkedEntity[];
+}
+
+// @public
+export interface InnerError {
+    code: ErrorCode;
+    innererror?: InnerError;
+    message: string;
+    target?: string;
+}
+
+// @public
+export interface LanguageInput {
+    // (undocumented)
+    countryHint?: string;
+    id: string;
+    // (undocumented)
+    text: string;
+}
+
+// @public
+export interface LinkedEntity {
+    dataSource: string;
+    id?: string;
+    language: string;
+    matches: Match[];
+    name: string;
+    url: string;
+}
+
+// @public
+export interface Match {
+    length: number;
+    offset: number;
+    score: number;
+    text: string;
+}
+
+// @public
+export interface MultiLanguageInput {
+    id: string;
+    language?: string;
+    text: string;
+}
+
+// @public (undocumented)
+export interface RecognizeEntitiesErrorResult extends TextAnalysisErrorResult {
+}
+
+// @public (undocumented)
+export interface RecognizeEntitiesOptions extends TextAnalyticsClientEntitiesRecognitionGeneralOptionalParams {
+}
+
+// @public (undocumented)
+export type RecognizeEntitiesResult = RecognizeEntitiesSuccessResult | RecognizeEntitiesErrorResult;
+
+// @public (undocumented)
+export interface RecognizeEntitiesResultCollection extends Array<RecognizeEntitiesResult> {
+    modelVersion: string;
+    statistics?: RequestStatistics;
+}
+
+// @public (undocumented)
+export interface RecognizeEntitiesSuccessResult extends TextAnalysisSuccessResult {
+    readonly entities: Entity[];
+}
+
+// @public (undocumented)
+export interface RecognizePiiEntitiesOptions extends TextAnalyticsClientEntitiesRecognitionPiiOptionalParams {
+}
+
+// @public
+export interface RequestStatistics {
+    documentsCount: number;
```

Comment:
Do we also plan to change to the singular here?

Reply:
This is what we've done in .NET.

If it's helpful, you can use the .NET API as a reference for this - I think I caught all instances of these (but if you find more please call them out!) https://apiview.azurewebsites.net/Assemblies/Review/b196a5e53fa2441f92e3b1accefa35e7

Reply:
Fixed these too! thanks for the pointer